### PR TITLE
Create/Edit multisig alias

### PR DESCRIPTION
**Pull request by Ayagoumi**
This PR introduces functionality that not only allows users to create a new multi-signature alias on the P-Chain, but also to edit existing multi-signature aliases.

Key Features:

 **Naming the multi-signature alias (memo):** Users can name their multi-signature alias for better identification and management. Existing aliases can also be renamed using the edit feature.

**Adding/Editing multi-signature owners:** This feature enables users to add multiple owners to a singleton wallet. Additionally, users can now edit the list of owners in case of changes.

**Editing the threshold:** Users can set a threshold to determine the minimum number of signatures needed to validate and process the transactions. This threshold can be adjusted as needed.

Additionally, this PR also handles error scenarios, including the following, same as for the create multisig feature

```
If the name of the multi-signature alias is too long (beyond 64 bytes)
If the number of owners is not sufficient or exceeds 128
If the threshold set exceeds the number of wallet owners
If the owner's wallet isn't a single wallet P-Chain Address
```

### CODE REVIEW

File: src/views/EditMultisigWallet.vue [hunk 1]
```diff
@@ -0,0 +1,752 @@
...
+    async issueMultisigTx() {
+        // @ts-ignore
+        let { dispatchNotification } = this.globalHelper()
+        const wallet = this.activeWallet
+        if (!wallet || !(wallet instanceof MultisigWallet))
+            return console.error('MultiSigTx::sign: Invalid wallet')
+        if (!this.pendingSendMultisigTX) return console.error('MultiSigTx::sign: Invalid Tx')
+        try {
+            await this.updateMultisigTxDetails()
+            await wallet.issueExternal(this.pendingSendMultisigTX?.tx)
```
Comment (aeddaqqa) [inline src/views/EditMultisigWallet.vue:546]: sometimes executing the transaction results in failure to decode client response: couldn't issue tx: alias credential isn't matching: input has more signers than expected

File: src/views/EditMultisigWallet.vue [hunk 2]
```diff
@@ -0,0 +1,752 @@
...
+            console.error('MultiSigTx::sign: Error', e)
+            dispatchNotification({
+                message: this.$t('notifications.execute_multisig_transaction_error'),
+                type: 'error',
+            })
+            throw e
+        }
+    }
+
+    async updateMultisigTxDetails() {
```
Comment (aeddaqqa) [inline src/views/EditMultisigWallet.vue:562]: After executing the transaction, displayed data are not properly update (i.e showing msig infos before the edit)

File: src/views/EditMultisigWallet.vue [hunk 3]
```diff
@@ -0,0 +1,752 @@
...
+                    <button
+                        @click="removeAddress(index)"
+                        class="circle delete-button"
+                        v-if="mode === 'EDIT'"
+                    >
+                        <fa icon="minus"></fa>
+                    </button>
+                </div>
+
+                <div class="add-new-address" v-if="addresses.length < 128 && mode === 'EDIT'">
```
Comment (aeddaqqa) [inline src/views/EditMultisigWallet.vue:57]: Adjust style

**Review by aeddaqqa — CHANGES_REQUESTED** [review]

File: src/views/EditMultisigWallet.vue (hunk 2 above)
Comment (Ayagoumi) [reply src/views/EditMultisigWallet.vue:562]: Resolved

File: src/views/EditMultisigWallet.vue (hunk 3 above)
Comment (Ayagoumi) [reply src/views/EditMultisigWallet.vue:57]: Resolved

File: src/views/EditMultisigWallet.vue (hunk 1 above)
Comment (Ayagoumi) [reply src/views/EditMultisigWallet.vue:546]: Resolved

File: src/views/CreateMultisigWallet.vue [hunk 4]
```diff
@@ -238,7 +247,7 @@ export default class CreateMultisigWallet extends Vue {
             )
 
             if (result) {
-                await this.updateMultisigAccount(result)
+                await this.addMultisigAccountToLocalStorage(result)
                 setTimeout(() => {
```
Comment (aeddaqqa) [inline src/views/CreateMultisigWallet.vue]: after creating msg you are updating the show alias state but not fetching the new alias from magellan -> requesting new alias is missing

File: src/views/EditMultisigWallet.vue [hunk 5]
```diff
@@ -0,0 +1,736 @@
...
+                            :disabled="mode !== 'EDIT' || pendingSendMultisigTX"
+                        />
+                        <CamInput
+                            class="msig-address-name"
+                            :placeholder="index === 0 ? 'My Address' : `Owner ${index + 1} Name`"
+                            v-model="address.name"
+                            :disabled="mode !== 'EDIT' || pendingSendMultisigTX"
+                        />
+                    </div>
+                    <button
```
Comment (aeddaqqa) [inline src/views/EditMultisigWallet.vue:48]: cant remove the first address

File: src/views/EditMultisigWallet.vue [hunk 6]
```diff
@@ -0,0 +1,736 @@
+<template>
+    <div class="msig-edit__container">
+        <h1>{{ $t('edit_multisig.title') }}</h1>
+        <div class="msig-create__form">
+            <div class="input-container">
+                <h3>{{ $t('create_multisig.name') }}</h3>
+                <div class="input-with-warning">
```
Comment (aeddaqqa) [inline src/views/EditMultisigWallet.vue:7]: space between the alert and teh title is wrong

**Review by aeddaqqa — COMMENTED** [review]
i have encoutred and issue with editing the msg the error that i got : 
-  "error": "failed to decode client response: couldn't issue tx: failed to verify alias or auth: output is unspendable"
how to reproduce 
- logged in with alias for a 3 addresses and threshold of 1
- delete 2 addresses
- edit the threshold to 3
- execute

**Review by aeddaqqa — APPROVED** [review]

**Pull request merged**